pkg/alertmanager/multitenant.go

```diff
@@ -864,7 +891,7 @@ func (am *MultitenantAlertmanager) alertmanagerFromFallbackConfig(ctx context.Co
 
 	level.Warn(am.logger).Log("msg", "no configuration exists for user; uploading fallback configuration", "user", userID)
 
-	// Upload an empty config so that the Alertmanager is no de-activated in the next poll
+	// Upload an empty config so that the Alertmanager is no de-activated in the next poll.
```

Comment:
[nit] This was probably just leftover from the last change

```suggestion
	// Upload an empty config so that the Alertmanager is no de-activated in the next poll
```